presto-spi/src/main/java/com/facebook/presto/spi/ConnectorSession.java

```diff
@@ -63,4 +64,6 @@ default boolean isReadConstraints()
     }
 
     WarningCollector getWarningCollector();
+
+    Optional<RuntimeStats> getRuntimeStats();
```

Comment:
Any reason this is made Optional ? The `Session` has this as **not** optional. I think its better if we follow the same types across the different session objects. wdyt ?

Reply:
Makes sense to change this to non-Optional given the implementation of session and RuntimeStats are on oss side.